### DISCUSSION TITLE
AvanteAsk is not mapped

### DISCUSSION THREAD

**Discussion by ofgirichardsonb** [yetone/avante.nvim#551] [Q&A]
I have installed a basic Neovim 0.10.1 installation with LazyVim. I have installed avante.nvim as per the instructions in `README.md`, and Lazy.nvim reports that it is loaded. But none of the key mappings like <leader>aa seem to be mapped! What am I missing? Running on MacOS/ARM64 if that matters.

**Comment by ofgirichardsonb** [comment]
It would seem that the answer was to build from source. First, set `build` to `make BUILD_FROM_SOURCE=true`. Second, remove the plugin from your `~/.local/share/nvim` directory, as it will not build again once it has been installed. Finally, install as usual. You will need `cargo` command available, which is included in the `rust` homebrew package.

**Reply by leoingru** [reply]
You need do `require('avante_lib').load()` 
This is what I have configed.
https://github.com/leoatchina/leovim/blob/master/conf.d/element/lua/aichat.lua#L109
